modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/page.jsp

```diff
@@ -27,7 +27,13 @@
 
 	dataSetDisplay.default(
 		{
-			actionParameterName: '<%= actionParameterName %>',
+
```

Comment:
This looks fine, but if we want to make sure `actionParameterName` is always a string, we could use `GetterUtil.getString()` in [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/init.jsp#L37).